src/schema/shared.ts

```diff
@@ -84,6 +84,13 @@ export const RawDateSchema = z.date().transform((i) => {
   return BigNumber.from(Math.floor(i.getTime() / 1000));
 });
 
+export const AirdropQuantitySchema = z
```

Comment:
I think we already have an alias for Price that is "Amount" - can you use that instead?

Reply:
not sure what you mean here, I added this because I realized the error message for airdrop when invalid quantity was wrong

Reply:
lets just call this `AmountSchema` and use it for the `Amount` type